pkg/storage/redis/redis_test.go

```diff
@@ -88,27 +88,29 @@ func TestDB(t *testing.T) {
 	if os.Getenv("GITHUB_ACTION") != "" && runtime.GOOS == "darwin" {
 		t.Skip("Github action can not run docker on MacOS")
 	}
-	redisTLSEnv := []string{
-		"ALLOW_EMPTY_PASSWORD=yes",
-		"REDIS_TLS_ENABLED=yes",
-		"REDIS_TLS_CERT_FILE=/tls/redis.crt",
-		"REDIS_TLS_KEY_FILE=/tls/redis.key",
-		"REDIS_TLS_CA_FILE=/tls/ca.crt",
+
+	cwd, err := os.Getwd()
```

Comment:
we might need to lookup the path of the source code. I'm not sure the current working directory is set consistently (ie if I run `go test ./...` wouldn't it be set to the project dir?)

Reply:
Good question.  It does work in the same directory as the tests and at the top level (eg make test), so it seems like it does the right thing WRT the execution location.  That might be special behavior for tests?

Reply:
Cool it seems to work.